arguments.go

```diff
@@ -27,15 +27,31 @@ import (
 	"io/ioutil"
 )
 
+// ArgReader is the interface for the arg2 and arg3 streams on an
+// OutboundCallResponse and an InboundCall
+type ArgReader interface {
+	io.ReadCloser
+}
```

Comment:
+1, we have no plans right now to add more methods, so this seems like a good idea.